### PR TITLE
changed hml alert for alert component and added some minor errorhandling

### PR DESCRIPTION
**Pull request by MGViereck**

### CODE REVIEW

File: src/views/SearchView.vue [hunk 1]
```diff
@@ -51,10 +55,24 @@ export default class SearchView extends Vue {
     }
 
     async onSearch(searchData: SearchData<void>) {
+        try {
```
Comment (dobschal) [inline src/views/SearchView.vue:58]: MGViereck I think it makes sense to set ```this.error=""``` when search is submitted to hide the last error message.

**Review by dobschal — CHANGES_REQUESTED** [review]

File: src/components/partials/IdentitySearch/IdentitySearch.vue [hunk 2]
```diff
@@ -149,10 +149,8 @@ export default class IdentitySearch extends Vue {
             this.selectedChainKey
         );
         if (!this.implementsPallet) {
-            // TODO: show nice error partial component instead of standard alert
-            alert(
-                "Sorry, the selected node is not available or does not implement the identity pallet"
-            );
+            const message = "Sorry, the selected node is not available or does not implement the identity pallet";
```
Comment (a-oertel) [inline src/components/partials/IdentitySearch/IdentitySearch.vue:152]: `this.implementsPallet = await this.store.dispatch(
            "IDENTITY_PALLET_EXISTS",
            this.selectedChainKey
        );
`
Should be in try catch. In the catch the message for the error should be "Sorry, the connection to the node could not be established"
and if it does not implement the identity pallet -> "Sorry, the selected node does not implement the identity pallet"

File: src/views/SearchView.vue (hunk 1 above)
Comment (MGViereck) [reply src/views/SearchView.vue:59]: is adapted in commit comment 1

**Review by a-oertel — CHANGES_REQUESTED** [review]

**Comment by a-oertel** [comment]
Also the listView has the IdentitySearch Partial and the error handling should be done there as well.

File: src/store/index.ts [hunk 3]
```diff
@@ -168,6 +173,10 @@ export const store = createStore({
             });
             context.commit("decrementBusyCounter");
             return implementsPallet;
+        },
+
+        async RESET_BUSY(context: ActionContext<StoreI, StoreI>): Promise<void> {
+            context.commit("resetBusyCounter");
```
Comment (a-oertel) [inline src/store/index.ts:179]: I actually think we should better use the "decrementBusyCounter" and call it DECREMENT_BUSY

File: src/views/SearchView.vue [hunk 4]
```diff
@@ -31,16 +32,22 @@ import { SearchData } from "@/interfaces/SearchData";
...
         Logo,
         IdentitySearch,
-        RecentSearch
+        RecentSearch,
+        Alert
     }
 })
 export default class SearchView extends Vue {
     store = useStore();
+    //queryString = window.location.search;
```
Comment (a-oertel) [inline src/views/SearchView.vue:47]: can we remove those comments?

Comment (MGViereck) [reply src/views/SearchView.vue]: Yes, and they are in the latest commit

File: src/store/index.ts (hunk 3 above)
Comment (MGViereck) [reply src/store/index.ts]: got changed in the last series of commits

File: src/components/partials/IdentitySearch/IdentitySearch.vue [hunk 5]
```diff
@@ -144,15 +144,19 @@ export default class IdentitySearch extends Vue {
...
+                this.selectedChainKey
             );
+            if (!this.implementsPallet) {
+                const message = "Sorry, the selected node is not available or does not implement the identity pallet";
+                this.$emit("error", message);
+            }
+        }
+        catch (e) {
+            const message = "Sorry, the connection to the node could not be established";
+            this.$emit("error", message);
```
[the remark this answers is not in the document]
Comment (MGViereck) [reply src/components/partials/IdentitySearch/IdentitySearch.vue:161]: is implemented

**Review by a-oertel — APPROVED** [review]

**Pull request merged**